src/utils/bytes.py

```diff
@@ -2,3 +2,7 @@ def from_hex_string_to_bytes(hex_string):
     if hex_string.startswith('0x'):
```

Comment:
add annotations to `from_hex_string_to_bytes` func

Reply:
Done